AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -841,6 +841,12 @@ - (void)setAccessibilityIdentifier:(NSString *)accessibilityIdentifier
   _setToViewOnly(accessibilityIdentifier, accessibilityIdentifier);
 }
 
+- (NSInteger)accessibilityElementCount
```

Comment:
Does this need to be defined in header as a bridge property? Otherwise the compiler may not allow it to be called directly on a node instance